<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qMRMLPlusServerLauncherTableView</class>
 <widget class="qMRMLWidget" name="qMRMLPlusServerLauncherTableView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plus Server Launcher Table</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="ShowRemoteCheckBox">
     <property name="text">
      <string>Show remote servers</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="AddServerButton">
       <property name="text">
        <string>Add server</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RemoveServerButton">
       <property name="text">
        <string>Remove server</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="PlusServerLauncherTable">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>State</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="AdvancedCollapsibleButton">
     <property name="text">
      <string>Advanced</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QFormLayout" name="ServerPanelLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="LocalControlLabel">
          <property name="text">
           <string>Local control:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="LocalControlCheckBox">
          <property name="toolTip">
           <string>If checked, the state button will start/stop the Plus server when clicked.</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="ConfigFileLabel">
          <property name="text">
           <string>Config file:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="qMRMLNodeComboBox" name="ConfigFileComboBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="nodeTypes">
           <stringlist>
            <string>vtkMRMLTextNode</string>
           </stringlist>
          </property>
          <property name="noneEnabled">
           <bool>true</bool>
          </property>
          <property name="renameEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="DescriptionLabel">
          <property name="text">
           <string>Description:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="LogLevelLabel">
          <property name="text">
           <string>Log level:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="LogLevelComboBox"/>
        </item>
        <item row="2" column="1">
         <widget class="QTextEdit" name="DescriptionTextEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qMRMLPlusServerLauncherRemoteWidget.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>qMRMLPlusServerLauncherTableView</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ConfigFileComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
